assets/javascript/feedback-tracking.js

```diff
@@ -24,22 +24,22 @@
     game: 'GAME',
     'landing-page': 'TOPIC',
     series: 'SERIES',
-    tags: 'TAG'
+    topic: 'TOPIC'
   };
 
   window._feedback = {};
 
   function feedbackTracker(rootElementId) {
     var widget = $('#' + rootElementId);
     var contentType = widget.data('item-type');
-    var secondaryTags = widget.data('item-tags');
+    var topics = widget.data('item-topics');
     var categories = widget.data('item-categories');
 
     window._feedback.title = widget.data('item-title');
     window._feedback.url = window.location.pathname;
     window._feedback.contentType = types[contentType] || contentType;
     window._feedback.categories = categories ? ('' + categories).split(',') : [];
-    window._feedback.secondaryTags = secondaryTags ? ('' + secondaryTags).split(',') : [];
+    window._feedback.topics = topics ? ('' + topics).split(',') : [];
```

Comment:
is this going to break feedback reports?

Reply:
No. These values are not currently utilised by the feedback reports.